include/deal.II/fe/fe_raviart_thomas.h

```diff
@@ -151,6 +150,11 @@ class FE_RaviartThomas
   memory_consumption() const override;
 
 private:
+  /**
+   * The polynomial space for the finite element.
+   */
+  const PolynomialsRaviartThomas<dim> polys;
```

Comment:
Why do you need this? Isn't the base class now storing an object of this type for you?

Reply:
Whoops! That may have been leftover from my early experiments.